<launch>

<!-- EKF -->
<node pkg="coconuts_odroid" type="SimpleMover" name="simple_mover" output="screen" >
	<param name="port" value="/dev/pts/27" />
	<param name="kp" value="0.0" />
	<param name="ki" value="0.0" />
	<param name="kd" value="0.0" />
</node>

<node pkg="usb_cam" type="usb_cam_node" name="stereo_left" output="screen">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="stereo_left" />
        <param name="io_method" value="mmap"/>
	<param name="camera_name" value="stereo_left" />
</node>

</launch>
